third_party/tcplp/Makefile.am

```diff
@@ -29,6 +29,7 @@
 include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 
 EXTRA_DIST                                    = \
+    lib                                         \
     tcplp.h                                     \
```

Comment:
I think you need to do the following to get `distcheck` to pass.
```suggestion
    tcplp.h                                     \
    lib/bitmap.h                                \
    lib/cbuf.h                                  \
    lib/lbuf.h                                  \
```

Reply:
Thanks! I've included this in my latest update.